docs/cmdlets-reference/set-cohesitycmdletconfig.md

```diff
@@ -72,6 +77,22 @@ Set-CohesityCmdletConfig -RefreshToken $true
 Enables the flag RefreshToken, the cmdlet framework would implicitly attempt to refresh the expired token.
 The user does not need to explicitly connect to the cluster post token expiry.
 
+### EXAMPLE 6
+```
+Set-CohesityCmdletConfig -OrganizationName org1
+```
+
+Set the organization (tenant).
+It would enable user to impersonate as tenant 'org1'.
```

Comment:
This would only impersonate admin user to Org user right ? , do we have support for user1@tenant1 and user2@tenant2 ?

Reply:
Cohesity does not have tenant based direct access.

Reply:
Can you experiment a bit more on the cluster, I remember having logged in with specific user with tenant id

Reply:
Let me check if its feasible to log with tenant id. Need to support v6.3 onward.

Reply:
The tenant can be accessed by logging as LOCAL\user1@tenant, therefore the cmdlets would work for the tenants as well.

Reply:
Reverting back the code changes for tenant based access.